src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const persons = document.body.querySelectorAll('li');
+
+function sortSalary() {
+  let previous = persons[0]
+    .getAttribute('data-salary')
+    .replace((/[,$]/g, ''), 10);
```

Comment:
The `replace` method is used incorrectly here. The second argument should be an empty string `''` to remove the characters, not `10`. This will cause a logical error in parsing the salary.